app/controllers/application_controller.rb

```diff
@@ -2,6 +2,7 @@ class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
 
   before_action :set_paper_trail_whodunnit
+  before_action :authenticate, only: :create unless Rails.env.production?
```

Comment:
Yes, I did not find how to do just one for an `action#create`
Now I understand what can be done by checking the controller